src/_imaging.c

```diff
@@ -2322,19 +2345,25 @@ static HPy Imaging_getextrema_impl(HPyContext *ctx, HPy self) {
     if (status) {
         switch (im_self->image->type) {
             case IMAGING_TYPE_UINT8:
-                return HPy_NULL;//HPy_BuildValue(ctx, "II", extrema.u[0], extrema.u[1]);
+                // return HPy_BuildValue(ctx, "BB", extrema.u[0], extrema.u[1]);
```

Comment:
I think this comment can go - there was an issue while benchmarking in which PyPy didn't like HPy_BuildValue, but it worked fine on CPython and GraalPy, so these HPy_BuildValue comments are just a legacy of then